### PR TITLE
Numbers API: Callback configuration endpoints added.

### PR DESCRIPTION
**Pull request by 650elx**

### CODE REVIEW

File: sinch/domains/numbers/__init__.py [hunk 1]
```diff
@@ -29,6 +30,14 @@
...
     GetNumberConfigurationResponse, ReleaseNumberFromProjectResponse
 )
+from sinch.domains.numbers.models.callbacks.responses import (
+    GetNumbersCallbackConfigurationResponse,
+    UpdateNumbersCallbackConfigurationResponse
+)
+
+from sinch.domains.numbers.models.callbacks.requests import (
+    UpdateNumbersCallbackConfigurationRequest
+)
```
Comment (Dovchik) [inline sinch/domains/numbers/__init__.py:40]: minor
```suggestion
from sinch.domains.numbers.models.callbacks.requests import UpdateNumbersCallbackConfigurationRequest
```

File: sinch/domains/numbers/endpoints/callbacks/get_configuration.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from sinch.core.models.http_response import HTTPResponse
+from sinch.domains.numbers.endpoints.numbers_endpoint import NumbersEndpoint
+from sinch.core.enums import HTTPAuthentication, HTTPMethods
+
```
Comment (Dovchik) [inline sinch/domains/numbers/endpoints/callbacks/get_configuration.py:4]: grouping import, is blank line necessary?

**Review by Dovchik — APPROVED** [review]
except minor

File: sinch/domains/numbers/endpoints/callbacks/get_configuration.py (hunk 2 above)
Comment (650elx) [reply sinch/domains/numbers/endpoints/callbacks/get_configuration.py]: fixed

**Pull request merged**